html/js/product-preferences.js

```diff
@@ -23,32 +24,18 @@ function get_user_product_preferences() {
 	return user_product_preferences;
 }
 
-function getFlavorFromHostname() { // get the flavor by the url 
-    var host = window.location.hostname;
+var type_product;
```

Comment:
Can you rename it to product_type? that's the variable name we use in other parts of the code